src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/AS_InbreedingCoeffUnitTest.java

```diff
@@ -0,0 +1,217 @@
+                makeG("s3", T, C, 7099, 2530, 7099, 3056, 0, 14931),
+                makeG("s4", Aref, T, 2530, 0, 7099, 366, 3056, 14931),
+                makeG("s5", T, T, 7099, 2530, 0, 7099, 366, 3056),
+                makeG("s6", Aref, T, 2530, 0, 7099, 366, 3056, 14931),
+                makeG("s7", T, T, 7099, 2530, 0, 7099, 366, 3056),
+                makeG("s8", Aref, T, 2530, 0, 7099, 366, 3056, 14931),
+                makeG("s9", T, T, 7099, 2530, 0, 7099, 366, 3056),
+                makeG("s10", Aref, T, 2530, 0, 7099, 366, 3056, 14931));
+
+        final AS_InbreedingCoeff testClass1 = new AS_InbreedingCoeff();
+        final double ICresult1 = testClass1.calculateIC(test1, T);
+        Assert.assertEquals(ICresult1, -0.4285714, DELTA_PRECISION, "Pass");
+        final double ICresult1b = testClass1.calculateIC(test1, C);
+        Assert.assertEquals(ICresult1b, -0.05263, DELTA_PRECISION, "Pass");
+
+        //make sure that hets with different alternate alleles all get counted
+        VariantContext test2 = makeVC("2", Arrays.asList(Aref,T,C),
+                makeG("s1", Aref, C, 4878, 1623, 11297, 0, 7970, 8847),
+                makeG("s2", Aref, T, 2530, 0, 7099, 366, 3056, 14931),
+                makeG("s3", Aref, T, 3382, 0, 6364, 1817, 5867, 12246),
+                makeG("s4", Aref, T, 2488, 0, 9110, 3131, 9374, 12505),
+                makeG("s5", Aref, C, 4530, 2006, 18875, 0, 6847, 23949),
+                makeG("s6", Aref, T, 5325, 0, 18692, 389, 16014, 24570),
+                makeG("s7", Aref, T, 2936, 0, 29743, 499, 21979, 38630),
+                makeG("s8", Aref, T, 6902, 0, 8976, 45, 5844, 9061),
+                makeG("s9", Aref, T, 5732, 0, 10876, 6394, 11408, 17802),
+                makeG("s10", Aref, T, 2780, 0, 25045, 824, 23330, 30939));
+
+        final AS_InbreedingCoeff testClass2 = new AS_InbreedingCoeff();
+        final double ICresult2 = testClass2.calculateIC(test2, T);
+        Assert.assertEquals(ICresult2, -0.666666, DELTA_PRECISION, "Pass");
+        final double ICresult2b = testClass2.calculateIC(test2, C);
+        Assert.assertEquals(ICresult2b, -0.111129, DELTA_PRECISION, "Pass");
+
+        //test the annotate method
+        final Map<String, Object> annots = new AS_InbreedingCoeff().annotate(null, test2, null);
+        Assert.assertEquals(annots.keySet(), Collections.singleton(GATKVCFConstants.AS_INBREEDING_COEFFICIENT_KEY), "annots");
+        Assert.assertEquals(annots.values().size(), 1, "size");
+        final String annotString = (String)annots.values().iterator().next();
+        final String[] split = annotString.split(",");
+        Assert.assertEquals(split.length, 2);
+        Assert.assertEquals(Double.parseDouble(split[0]), -0.666666, DELTA_PRECISION, "ic");
+        Assert.assertEquals(Double.parseDouble(split[1]), -0.111129, DELTA_PRECISION, "ic");
+
+        if (false) { //TODO reenable when AS_InbreedingCoeff uses founders
```

Comment:
why? using if false is far superior IMO - checks the the code compiles and gets refactored with other code.